GI01SUMO/GI01SUMO_R00008_ingest.csv

```diff
@@ -59,10 +59,10 @@ mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/w
 mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/cg_data/dcl11/imm/pco2w3/*.DAT,GI01SUMO-RII11-02-PCO2WC053,recovered_host,Available,
 #mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/instruments/dcl11/imm/PCO2W_snC0121/SAMI*.txt,GI01SUMO-RII11-02-PCO2WC053,recovered_inst,Not Expected,No comms with instrument upon recovery
 mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/cg_data/dcl11/imm/phsen1/*.DAT,GI01SUMO-RII11-02-PHSENE041,recovered_host,Available,
-#mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/instruments/dcl11/imm/PHSEN_snP0138/SAMI*.txt,GI01SUMO-RII11-02-PHSENE041,recovered_inst,Not Expected,No comms with instrument upon recovery
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/instruments/dcl11/imm/PHSEN_snP0138/SAMI*.txt,GI01SUMO-RII11-02-PHSENE041,recovered_inst,Available,
 mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/cg_data/dcl11/imm/phsen2/*.DAT,GI01SUMO-RII11-02-PHSENE042,recovered_host,Available,
 mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/instruments/dcl11/imm/PHSEN_snP0139/SAMI*.txt,GI01SUMO-RII11-02-PHSENE042,recovered_inst,Available,
-#mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/cg_data/cpm1/irid2shore/cpm_status*.txt,GI01SUMO-SBC11-00-CPMENG000,recovered_host,Expected,Data could not be downloaded at sea but are still expected
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/GI01SUMO/R00008/cg_data/cpm1/irid2shore/cpm_status*.txt,GI01SUMO-SBC11-00-CPMENG000,recovered_host,Available,

```

Comment:
looks good; confirmed w/ JB that still pending C0121 instrument data recovery attempt so that line remains commented out for now.